tests/model_tests.py

```diff
@@ -105,3 +105,9 @@ def test_grains_dict(self):
         self.assertEquals(d.get('day').function, 'DATE({col})')
         self.assertEquals(d.get('P1D').function, 'DATE({col})')
         self.assertEquals(d.get('Time Column').function, '{col}')
+
+    def test_get_timestamp_expression(self):
+        tbl = self.get_table_by_name('birth_names')
+        ds_col = tbl.get_column('ds')
+        sqla_literal = ds_col.get_timestamp_expression(None)
```

Comment:
Can we add a test thats passing a value into `get_timestamp_expression()` to make sure we are returning the proper `literal_col`

Reply:
Or just add one test per database:
https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures